.travis.yml

```diff
@@ -31,6 +31,7 @@ matrix:
       env:
         - DEPENDS="cython numpy scipy matplotlib h5py nibabel cvxopt scikit_learn"
 before_install:
+    - apt-get update && apt-get install -y python-vtk
     - source tools/travis_tools.sh
     - virtualenv --python=python venv
```

Comment:
Will this allow me to install with apt-get?

On Tue, Oct 13, 2015 at 5:52 PM, Matthew Brett <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/nipy/dipy/pull/732#discussion_r41945276>:
>
> > @@ -31,6 +31,7 @@ matrix:
> >        env:
> >          - DEPENDS="cython numpy scipy matplotlib h5py nibabel cvxopt scikit_learn"
> >  before_install:
> > +    - apt-get update && apt-get install -y python-vtk
> >      - source tools/travis_tools.sh
> >      - virtualenv --python=python venv
>
> I think you'll need virtualenv --system-site-packages --python=python venv
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/732/files#r41945276>.
>